### PR TITLE
Assistant Parameter Fixes

### PR DESCRIPTION
**Pull request by marcbonnici**

### CODE REVIEW

File: wa/commands/show.py [hunk 1]
```diff
@@ -91,13 +91,7 @@ def get_rst_from_target(target):
...
-    conn_params_rst = get_params_rst(target.conn_params)
-    if conn_params_rst:
-        text += underline('\nconnection parameters', '~') + conn_params_rst
-    platform_params_rst = get_params_rst(target.platform_params)
-    if platform_params_rst:
-        text += underline('\nplatform parameters', '~') + platform_params_rst
-    target_params_rst = get_params_rst(target.target_params)
-    if target_params_rst:
-        text += underline('\nconnection parameters', '~') + target_params_rst
+    text += get_params_rst(target.conn_params)
```
Comment (setrofim) [inline wa/commands/show.py]: I's probably worth having a single heading for all three, e.g. "device parameters" to reflect the name of the agenda section where they go.

Comment (marcbonnici) [reply wa/commands/show.py]: Updated

**Comment by marcbonnici** [comment]
Currently broken, do not merge.

**Comment by marcbonnici** [comment]
Fixed

**Pull request merged**